src/classes/SObjectFieldWrapper.cls

```diff
@@ -0,0 +1,29 @@
+public class SObjectFieldWrapper {
```

Comment:
I like the idea of this class a lot. I think it will help with...
  * Filtering on parent fields (obviously)
  * Including parent fields in a query's select statement (future enhancement)
  * Including parent fields in an aggregate query's group by statement (future enhancement)
  * Order by parent fields (sounds like you've already got that on your end, but shouldn't be too bad to add to nebula as a future enhancement)